frameworks/cassandra/universe/config.json

```diff
@@ -91,6 +91,11 @@
           "description": "The Marathon-style placement constraint string controlling node placement",
           "default": ""
         },
+        "virtual_network":{
```

Comment:
Curious about not making the overlay network membership arbitrarily configurable: do we expect that the usage is going to be either `dcos` overlay or nothing?